operators-in-select-statements/answers.md

```diff
@@ -0,0 +1,194 @@
+* Write statements to select the following pieces of data.
+* 
+### 3.1 Display the name, gender, and age of all dogs that are part Labrador.
+
+```SELECT name, gender, age from dogs WHERE breed LIKE '%labrador%';```
+
+Result:
+
+|  name  | gender | age | 
+|--------|--------|-----|
+| Boujee | F      |   3 |
+| Marley | M      |   0 |
+
+
+
+### 3.2 Display the ids of all dogs that are under 1 year old.
+
+```SELECT id from dogs WHERE age < 1; ```
+
+Result:
+|  id   | 
+|-------|
+| 10002 |
+| 10004 |
+
+### 3.3 Display the name and age of all dogs that are female and over 35lbs.
+
+```SELECT name, age FROM dogs WHERE gender = 'F' AND weight > 35;```
+
+Result:
+
+|  name  | age |
+|--------|-----|
+| Boujee |   3 |
+
+### 3.4 Display all of the information about all dogs that are not Shepherd mixes.
+```SELECT * FROM dogs WHERE breed NOT LIKE '%shepherd%';```
+
+Result:
+|  id   |   name    | gender | age | weight |       breed        | intake_date | in_foster  |  
+|-------|-----------|--------|-----|--------|--------------------|-------------|------------|
+| 10001 | Boujee    | F      |   3 |     36 | labrador poodle    | 2017-06-22  | null       |
+| 10002 | Munchkin  | F      |   0 |      8 | dachsund chihuahua | 2017-01-13  | 2017-01-31 |
+| 10004 | Marley    | M      |   0 |     10 | labrador           | 2017-05-04  | 2016-06-20 |
+| 10006 | Marmaduke | M      |   7 |    150 | great dane         | 2016-03-22  | 2016-05-15 |
+| 10007 | Rosco     | M      |   5 |    180 | rottweiler         | 2017-04-01  | null       |
+
+### 3.5 Display the id, age, weight, and breed of all dogs that are either over 60lbs or Great Danes.
+Intake teams typically guess the breed of shelter dogs, so the breed column may have multiple words (for example, "Labrador Collie mix").
+
+```
+SELECT id, age, weight, breed FROM dogs WHERE breed='great dane' OR weight > 60;
+```
+
+Result:
+
+|  id   | age | weight |   breed    |
+|-------|-----|--------|------------|
+| 10006 |   7 |    150 | great dane |
+| 10007 |   5 |    180 | rottweiler |
+
+
+## 4.From the cats table below, what pieces of data would be returned from these queries?
+
+```SELECT name, adoption_date FROM cats;```
+| name     | adoption_date |
+| -------- |---------------|
+| Mushi    | 03-22-2016    |
+| Seashell |               |
+| Azul     | 04-17-2016    |
+| Victoire |  09-01-2016   |
+| Nala     |               |
+
+
+```SELECT name, age FROM cats;```
+| name     |age |
+| -------- |----|
+| Mushi    | 1  |
+| Seashell | 7  |
+| Azul     | 3  |
+| Victoire | 7  |
+| Nala     | 1  |
+
+## 5. From the cats table, write queries to select the following pieces of data.
+### 5.1 Display all the information about all of the available cats.
+```
+SELECT * from cats;
+```
+### 5.2 Choose one cat of each age to show to potential adopters.
+```
+SELECT * FROM cats WHERE gender='M';
+```
+(This just happens to work, because there are 3 male cats, of age 1, 3 and 7.)
```

Comment:
Clever, I like it. In case you are curious, there is a way to do this relatively abstractly:

```sql
SELECT *
FROM cats c
WHERE id IN (
  SELECT id
  FROM cats d
  WHERE c.age = d.age
  LIMIT 1
);
```

However, that requires features not available in all SQL implementations, especially not older ones. When you work through sub-queries, you might be able to make more sense of that.